addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1191,22 +1192,41 @@ def _send_transient_message(self, partner_to, content):
         })
 
     def execute_command_help(self, **kwargs):
-        partner = self.env.user.partner_id
-        if self.channel_type == 'channel':
-            msg = _("You are in channel <b>#%s</b>.", html_escape(self.name))
+        if self.channel_type == "channel":
```

Comment:
I guess all the '' -> "" are unwanted diff and in python, '' is preferred when not dealing with string-related

Reply:
`"` is now preferred for everything, but it can be reverted in place where the line is not changed in current diff